modules/apps/headless/headless-builder/headless-builder-test/src/testIntegration/java/com/liferay/headless/builder/model/listener/test/APISchemaRelevantObjectEntryModelListenerTest.java

```diff
@@ -118,6 +129,40 @@ public void test() throws Exception {
 			"An API schema must be an existing object definition.",
 			jsonObject.get("title"));
 
+		JSONAssert.assertEquals(
```

Comment:
I think this test scenario should be in `APIEndpointRelevantObjectEntryModelListenerTest`, because you are using the `headless-builder/endpoint`, what do you think?

Reply:
will analyse now and move if needed!